modelci/finetuner/README.md

```diff
@@ -1,7 +1,84 @@
 # Finetuner
 
-## Fine tune for Computer Vision
-Tune ResNet18 on CIFAR10 for 25 epochs:
-```shell script
-python computer_vision_fune_tuning.py
+\[Experimental\] Finetuner is a tiny module of ModelCI for easy fine-tuning pre-trained model on a new dataset. 
+
+Finetuner **packs the training job** with specified Transfer Learning algorithm, dataset, and pre-trained model 
+written on various ML framework (e.g. PyTorch). Such training jobs are **managed and schedules** by a coordinator. 
+We are to support various training environment, such as local machine, private cluster, and public cloud platform. 
+
+We leverage [PyTorch Lihgtning](https://github.com/PyTorchLightning/pytorch-lightning) as the training backend for
+PyTorch models.
+
+## Structure
+```text
+├── coordinator           # manages and schedules training jobs
+├── torch_data_module.py  # creates data loader from dataset
```

Comment:
pytorch_dataloader.py

Reply:
`pytorch_datamodule` is better. It is named from `torch_lightning.LightningDataModule`.